kitsune/search/config.py

```diff
@@ -16,40 +16,365 @@ def get_index_name(index_name):
 USER_INDEX_NAME = get_index_name("user")
 FORUM_INDEX_NAME = get_index_name("forum_document")
 
-ES_LOCALE_ANALYZERS = {
-    "ar": "arabic",
-    "bg": "bulgarian",
-    "cs": "czech",
-    "fa": "persian",
-    "hi-IN": "hindi",
-    "id": "indonesian",
-    "ja": "cjk",
-    "pl": "polish",
-    "th": "thai",
-    "zh-CN": "chinese",
-    "zh-TW": "chinese",
+# # Locales that uses snowball tokenizer
+# ES_SNOWBALL_LOCALES = {
+#     "eu": "Basque",
+#     "ca": "Catalan",
+#     "da": "Danish",
+#     "nl": "Dutch",
+#     "en-US": "English",
+#     "fi": "Finnish",
+#     "fr": "French",
+#     "de": "German",
+#     "hu": "Hungarian",
+#     "it": "Italian",
+#     "no": "Norwegian",
+#     "pt-BR": "Portuguese",
+#     "pt-PT": "Portuguese",
+#     "ro": "Romanian",
+#     "ru": "Russian",
+#     "es": "Spanish",
+#     "sv": "Swedish",
+#     "tr": "Turkish",
+# }
```

Comment:
I've used the default [`stemmer`](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-stemmer-tokenfilter.html) for each language below, because that was the default in elastic.

We could use these snowball locales instead, with the [`snowball`](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-snowball-tokenfilter.html) tokenfilter. In fact some of these snowball stemmers are available through the `stemmer` tokenfilter, and some are even the default.

I haven't been able to find much in the way of documentation about which stemmers are better than others.